requirements.txt

```diff
@@ -1,3 +1,3 @@
 fonttools==3.28.0
-ufoLib==2.2.2
+ufoLib==2.2.5
```

Comment:
but I haven't tagged it yet...

Reply:
please wait until tomorrow, thanks

Reply:
oh, thats it, I wait until your done :)

thanks